src/org/gk/gkCurator/GKCuratorFrame.java

```diff
@@ -384,9 +384,9 @@ else if (e.getPropertyName().equals("switchType")) {
 	                    GKInstance instance = (GKInstance) e.getNewValue();
 	                    GKSchemaClass oldCls = (GKSchemaClass) e.getOldValue();
 	                    schemaView.switchedType(oldCls, instance);
-	                    eventView.switchedType(oldCls, instance);
-	                    entityView.switchedType(oldCls, instance);
```

Comment:
Right there is an extra-call of entityView.switchedType(), which should be deleted. However, please keep one call. Not delete both of these calls!